CapuaEnvironment/MemoryArray/MemoryArray.py

```diff
@@ -54,7 +53,7 @@ def __init__(self):
         for address in range(MEMORY_START_AT, MEMORY_END_AT):  # MEMORY_END_AT is NOT inclusive!!!
             # Create a cell for each memory address available default permission is
             # hardcoded to all access. This will be adjusted after Capua is "booted up".
-            mc = MemoryCell(permission=0b111, value=MEMORY_CELL_INITIAL_VALUE)
+            mc = MEMORY_CELL_INITIAL_VALUE
             self._memoryCellArray.append(mc)
```

Comment:
While you are touching this up, you can simplify this to:
```python3
def __init__(self):
         """
         This will initialize the whole memory cell array. No parameters required, we want Capua
         environment to be configurable by the user with minimal code change.
         Configuration for memory cell array are in Configuration.Configuration
         """
         # Create a cell for each memory address available default permission is
         # hardcoded to all access. This will be adjusted after Capua is "booted up".
         self._memoryCellArray = [
              MEMORY_CELL_INITIAL_VALUE
              for x in range(MEMORY_START_AT, MEMORY_END_AT)
         ]
```
This also passes my pep8

Reply:
Good call